src/cli.rs

```diff
@@ -210,7 +212,19 @@ pub struct VisualsCliOptions {
     ///
     /// Replaces language chips with Nerd Font icons
     #[arg(long)]
-    pub nerd_fonts: bool,
+    pub nerd_fonts: Option<bool>,
+}
+
+#[derive(Clone, Debug, Args, PartialEq, Eq)]
+#[command(next_help_heading = "CONFIG")]
+pub struct ConfigCliOptions {
+    /// Path to the config file
+    #[arg(long, value_hint = ValueHint::FilePath)]
+    pub config_path: Option<PathBuf>,
+    /// Creates a default config file
+    /// Writes to $XDG_CONFIG_HOME/onefetch/config.toml but can be overridden with --config-path
```

Comment:
This is only true for Linux. If you want to get fancy, you can change these docs with conditional compilation. Otherwise, you can just generically say something like `onefetch/config.toml in your home directory`.